### ISSUE TITLE
UI: data focus ring treatment should be uniform

### ISSUE THREAD

**Issue by tierninho** [desktop/desktop#5802]
The data focus ring is inconsistent in Desktop and should be uniform.

Version 1 looks better, as the entire data focus ring can be seen. Although this is not a tab like below, the UI treatment is visible and clear.

<img width="281" alt="screen shot 2018-10-02 at 11 43 00 am" src="https://user-images.githubusercontent.com/14828183/46369693-89f80080-c638-11e8-89d8-037f89444ce3.png">

Version 2 has a data focus ring on the outside of the tab. It hangs off the modal, is tucked under any tabs or certain areas next to it, and will overlap darker areas of the app, making it hard to see. At minimum, we should bring the data focus ring into the foreground if a tab is in focus.

<img width="488" alt="screen shot 2018-10-02 at 11 42 39 am" src="https://user-images.githubusercontent.com/14828183/46369692-895f6a00-c638-11e8-9903-537a373e268a.png">
<img width="272" alt="screen shot 2018-10-02 at 11 43 46 am" src="https://user-images.githubusercontent.com/14828183/46369694-89f80080-c638-11e8-9cfa-92d68ce4b608.png">

@desktop/design If this is by design or not worth fixing, please close.

1.4.2-beta0, amc

**Comment by donokuda** [comment]
Version 2 wasn't by design. 😅 

If someone wanted to take a crack at fixing the tab bar, a good pull request to reference would be https://github.com/desktop/desktop/pull/1287

I believe https://github.com/desktop/desktop/blob/master/app/styles/ui/_tab-bar.scss would be a good starting place for fixing this.

**Comment by GENERALGUBERNATOR1** [comment]
is.
